src/figuresClasses.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.Rectangle = exports.Circle = exports.Triangle = void 0;
+exports.getInfo = getInfo;
+var Triangle = /** @class */ (function () {
+    function Triangle(color, a, b, c) {
+        this.color = color;
+        this.a = a;
+        this.b = b;
+        this.c = c;
+        this.shape = 'triangle';
+        var sides = [a, b, c];
+        if (sides.some(function (s) { return s <= 0; })) {
+            throw new Error('Invalid sides');
```

Comment:
[CODE STYLE] - If you throw an error try to provide its detailed description in error message. Consider providing more context to the error message, such as 'Triangle side lengths must be greater than zero.'